.gitignore

```diff
@@ -3,6 +3,13 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+# build output
+**/dist/**
+**/lib/**
+*.tgz
+!ts-starter/resources/*.tgz
+!ts-starter-complete-cotta/resources/*.tgz
```

Comment:
Make sure to remove these once you import f rom the publish tasks

Reply:
Removed